catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -762,10 +764,36 @@ public void createCollisionPolygons() {
 
 	public class PenConfiguration {
 		public boolean penDown = false;
-		public float penSize = BrickValues.PEN_SIZE;
+		public float penSize = getDefaultPenSize();
 		public Color penColor = BrickValues.PEN_COLOR;
 		public PointF previousPoint = null;
 		public boolean stamp = false;
+		public float screenRatio = 1.0f;
+
+		private float getDefaultPenSize() {
```

Comment:
I didn't look at the specifics of the calculation, but the method does more than getDefaultPenSize.
It also implicitly sets the screenRatio.

I think it would be better to have it calculate and return the default screen ratio, call it at declaration of the screenRati member and then declare the penSize member from that.
Like this:
public float screenRatio = calculateScreenRatio()
public float penSize = 1.0f * screenRatio

Reply:
if updated the PR.
As suggested I'm now calculating first the screenRatio in a seperate function and then I'm calculating the pensize in the getDefaultPenSize function. (can't use BrickValues.PEN_SIZE since this variable gets calculated :) )

Also i've already rebased the PR since there was a conflict upcoming